lib/developer_portal/lib/liquid/drops/payment_gateway.rb

```diff
@@ -6,11 +6,16 @@ def initialize(account)
         @account = account
       end
 
-      desc "Returns whether current payment gateway is Braintree."
+      desc "Returns whether current payment gateway is authorize.Net."
```

Comment:
this is wrong

Reply:
It is, although it was like that originally :sweat_smile:  https://github.com/3scale/porta/commit/a31cd4011bba74b7bbdb152db549980f1fd5f35c#diff-0402b588da3aab79767114e1baea7824c5dacc66a00ae9f73492fe612fb03494